### PR TITLE
[nats-streaming] Adding arm32v6 architecture

### PR DESCRIPTION
**Pull request by kozlovic**
Added the arm32v6 arch.

### CODE REVIEW

File: library/nats-streaming [hunk 1]
```diff
@@ -1,26 +1,28 @@
 Maintainers: Ivan Kozlovic <ivan@synadia.com> (@kozlovic)
 GitRepo: https://github.com/nats-io/nats-streaming-docker.git
-GitCommit: 48f1a5bd7bc869f50d1379035fa65505196abc28
+GitCommit: 241ef65433ca7fa0042d6015b3d1ffd191a185ec
 
 Tags: 0.8.0-beta-linux, linux
 SharedTags: 0.8.0-beta, latest
 Architectures: amd64, arm32v7, arm64v8
```
Comment (yosifkit) [inline library/nats-streaming]: This line needs an `arm32v6` :wink:

Comment (kozlovic) [reply library/nats-streaming]: Sorry... will update soon.

**Comment by kozlovic** [comment]
@yosifkit Thanks again for the catch, and I hope it is fine now.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 536f898..09a1893 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,4 +1,5 @@
 nats-streaming:linux @ amd64
+nats-streaming:linux @ arm32v6
 nats-streaming:linux @ arm32v7
 nats-streaming:linux @ arm64v8
 nats-streaming:nanoserver @ windows-amd64
```

</details>

**Comment by alexellis** [comment]
Thank you @kozlovic that's really helpful and thanks for a quick turn-around on it too. 👍

**Pull request merged**